lib/fizzy/execute.cpp

```diff
@@ -214,8 +226,13 @@ Instance instantiate(const Module& module)
 
 execution_result execute(Instance& instance, FuncIdx function, std::vector<uint64_t> args)
 {
-    // TODO: handle the case when function index points to import
-    const auto& code = instance.module.codesec[function];
+    if (function < instance.imported_functions.size())
```

Comment:
Can you rename `function` to `func_idx`? This name is confusing.

Reply:
But please do it on its own commit :)